src/en/anchira/src/eu/kanade/tachiyomi/extension/en/anchira/AnchiraDto.kt

```diff
@@ -0,0 +1,57 @@
+package eu.kanade.tachiyomi.extension.en.anchira
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ListEntry(
+    @SerialName("id") val id: Int,
+    @SerialName("key") val key: String,
```

Comment:
Are these `@SerialName` decorators needed? They seem like the same as the property names.

Reply:
These were leftovers from when they had a different name.